style.scss

```diff
@@ -79,7 +96,7 @@ body {
     cursor:pointer;
 }
 
-.todo-item:hover .todo-item__destroy {
+.todo-item:hover .todo-item__destroy, .todo-item:focus .todo-item__destroy {
```

Comment:
Why do you need to set this to display block for focus, hover.

These are some weird nestings and approaches which should be avoided.

Reply:
I use CSS for toggling display of delete btn when mouse hovers over a particular todo display. (Toggle between display: none and display: block)

Another way to have done this would be complete in JS (adding/removing elements from DOM)

Let me know your preference.

Reply:
The hover was something I was experimenting with to allow the todo item and delete button to be toggled to. Didn't work, that :focus selector is leftover, forgot to remove. Maybe we can have discussion around how to manage keyboard focus.

Reply:
Another option: use completely different templates for the normal and editing states. track editing flag in view and render different template accordingly. let's discuss

Reply:
Do it with handlebars templates. When you enter edit state, a render happens, and there is an `if` that occurs to determine if it should render a label or an input.